README.md

```diff
@@ -369,7 +369,7 @@ Contributions are welcome. Please take a quick look at the [contribution guideli
 
 ## Routing
  * [crouter](https://github.com/jreinert/crouter) - A standalone router
- * [orion](https://github.com/obsidian/orion) - A minimal, rails'esk routing library.
+ * [orion](https://github.com/obsidian/orion) - A minimal, rails'esque routing library.
```

Comment:
I'm not a native english speaker, but isn't the correct _rails-esque/rails-ish_?.

Reply:
yeah, it's a `-` not a `'`